docs/examples/python/smessage_ssocket_sv.py

```diff
@@ -46,6 +46,7 @@ def close(self):
         return self.socket.close()
 
     def bind(self, param):
+        self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
it (SO_REUSEADDR) helps us avoid errors with sequential tests which uses same port?

Reply:
Yeah. For Python especially because there are a lot of variations. Without SO_REUSEADDR we’d be running into kernel timeouts for reusing ports that activate even if you close the listening socket in a orderly manner.